### PR TITLE
Adding detailed documentation on how to run conformance tests with bazel

### PR DESCRIPTION
**Pull request by mkruskal-google**

### CODE REVIEW

File: conformance/README.md [hunk 1]
```diff
@@ -40,15 +40,43 @@ tests using similar patterns.  You can either use Bazel to run the
...
 the conformance test executables outside C++.  So if you aren't using Bazel
 you'll need to create the executable you pass to `conformance_test_runner` via
 some alternate build system.
 
+While we plan to model all our supported languages more completely in Bazel,
+today some of them are a bit tricky to run.  Below is a list of the commands
+(and prerequisites) to run each language's conformance tests.
+
+Java:
+    $ bazel test //java/core:conformance_test //java/lite:conformance_test
```
Comment (Logofile) [inline conformance/README.md:53]: Add a link break between, and surround the command with backticks to get the formatting to match the previous section. Apply to all of the languages.

File: conformance/README.md [hunk 2]
```diff
@@ -40,15 +40,43 @@ tests using similar patterns.  You can either use Bazel to run the
...
+    $ which dotnet || echo "You must have dotnet installed!"
+    $ bazel test //csharp:conformance_test \
+    $   --action_env=DOTNET_CLI_TELEMETRY_OPTOUT=1 --test_env=DOTNET_CLI_HOME=~ \
+    $   --action_env=DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
+
+Objective-c (Mac only):
+    $ bazel test //objectivec:conformance_test --macos_minimum_os=10.9
+
+Ruby:
+    $ [[ $(ruby --version) == "ruby"* ]] || echo "Select a c ruby!"
```
Comment (Logofile) [inline conformance/README.md:71]: Should that be "C Ruby"?

File: conformance/README.md [hunk 3]
```diff
@@ -40,15 +40,43 @@ tests using similar patterns.  You can either use Bazel to run the
...
+    $ bazel test //ruby:conformance_test --define=ruby_platform=c \
+    $   --action_env=PATH --action_env=GEM_PATH --action_env=GEM_HOME
+
+JRuby:
+    $ [[ $(ruby --version) == "jruby"* ]] || echo "Switch to java ruby!"
```
Comment (Logofile) [inline conformance/README.md:76]: "Java Ruby"

File: conformance/README.md (hunk 1 above)
Comment (mkruskal-google) [reply conformance/README.md]: The backtick doesn't seem to do anything for the formatting, but I'll add the line break and try to make these more copy-pastable

Comment (Logofile) [reply conformance/README.md:54]: Intro text on one line, a blank line, a line with three backticks, the lines with code, a line with three backticks, and then another blank line should work. This is according to https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks

Comment (mkruskal-google) [reply conformance/README.md:54]: Ah gotcha, this isn't really code though.  It's commands to be copy/pasted.  The way it is now seems more consistent with our other command snippets, how's this look?

Comment (mkruskal-google) [reply conformance/README.md:54]: For some background: Sandy was having a really hard time figuring out how to actually run these conformance tests, some of which are still in a very intermediate state.  So I just wanted to have something to point people at before I forget the magic commands :)

Comment (Logofile) [reply conformance/README.md:54]: LGTM

**Review by Logofile — APPROVED** [review]

**Pull request merged**